Tests/RxSwift/filterMapTests.swift

```diff
@@ -50,18 +55,49 @@ class FilterMapTests: XCTestCase {
         subject.on(.next(1))
         subject.on(.next(2))
         subject.on(.next(3))
-        subject.on(.error(FilterMapTestError.error))
+        subject.on(.error(testError))
```

Comment:
I kinda prefer the old way, where the operator tests had an error, so it's contained.

Reply:
Which pros does this approach have? I can see that contributor have to create his own dummy error in each class

Reply:
Contained tests, some operators may need errors that are more complex/specific, etc.

Reply:
But unless this error needed why not to use general dummy error?

Reply:
It's just the way things currently happen inside every operator test in the repo. If you look at other tests in the repo you'll notice some have their own implementations and I'm not sure it's worth moving everything over. I don't super care to be honest.

Reply:
Me too. So i would prefer leave general dummy error for future test simplification, if you don't mind

Reply:
Sure.